src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -143,13 +188,48 @@ function renderBranch(
               }
             />
           );
+        break;
+      }
+      case 'DocumentReference': {
+        const documentReference = documentation.resource as fhir.DocumentReference;
+        const subject = documentReference.subject;
+        subject &&
```

Comment:
I see this is following along with the style from above, but in general this approach of `object && thing-to-do-if-object-not-null()` is not something we should be doing in pure JS code. Ideally these should be `if(object) { thing-to-do-if-object-not-null() }`. Where we would use the `&&` approach is in JSX because you can't do `if`  blocks within JSX.

Reply:
I will go and update this throughout ExpandedNode.tsx